external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/package.scala

```diff
@@ -16,9 +16,25 @@
  */
 package org.apache.spark.sql
 
+import java.util.concurrent.TimeUnit
+
 import org.apache.kafka.common.TopicPartition
 
+import org.apache.spark.internal.config.ConfigBuilder
+
 package object kafka010 {   // scalastyle:ignore
   // ^^ scalastyle:ignore is for ignoring warnings about digits in package name
   type PartitionOffsetMap = Map[TopicPartition, Long]
-}
+
+  private[spark] val PRODUCER_CACHE_TIMEOUT =
```

Comment:
Can these be more private, to `kafak010`? actually I realize now that the package statement at the top is wrong! you could fix that here.

Reply:
@hehuiyuan I mean that this file is in directory `org/apache/spark/sql/kafka010/` so its package should be `org.apache.spark.sql.kafka010`, but that's not what is declared at the top.

Reply:
![kafka010](https://user-images.githubusercontent.com/18002496/56344297-a5e77c00-61ef-11e9-864d-69661285e116.jpg)

Reply:
This is a package object, so the top is `package org.apache.spark.sql`